### PR TITLE
documenting rake tasks and optimizations added in #133

### PR DESCRIPTION
**Pull request by filmaj**
## Purpose of this pull request

documenting rake tasks and optimizations added in #133

## Affected documentation pages

`README.md`

## Additional information

#133

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: README.md [hunk 1]
```diff
@@ -124,14 +124,29 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
 > ***TIP***
 > Leave the serve terminal open and running. Every time you save changes to a file, it automatically regenerates the site so you can test the output immediately. Changing the `_config.yml` file requires a fresh build. Using the `--incremental` option limits re-builds to posts and pages that have changed.
 
-### To minimize build time locally:
+### Minimizing Build Time
+
+#### Building Specifc Editions
 
 You can speed the local build time up by only building one of the three versions that the full `rake preview` command builds. These are:
```
Comment (hickskenh) [inline README.md]:
```suggestion
You can speed the local build time up by only building one of the three versions that the full `rake preview` command builds. Building individual versions also enables incremental builds when you save a changed document, thus updating your preview. Incremental builds are disabled when `rake preview` is used to build all versions. To build a specific version, use one of the following commands:
```

File: README.md [hunk 2]
```diff
@@ -124,14 +124,29 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
...
 - `rake preview:ee`: Preview the Commerce edition locally
 
+#### Customizing Local Config
+
+By creating your own `_config.local.yml` file, you can customize the [Jekyll
+config](https://jekyllrb.com/docs/configuration/options/) to suit your needs.
+
+##### Disable Link Checking
+
+By default, every build ensures every link in every changed page, which can
```
Comment (hickskenh) [inline README.md]:
```suggestion
By default, `check_links` is enabled. This option verifies every link in every project file, which can
```

File: README.md [hunk 3]
```diff
@@ -124,14 +124,29 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
...
+significantly lower build times when iterating many changes quickly. You can
```
Comment (hickskenh) [inline README.md]:
```suggestion
significantly increase build times when iterating many changes quickly. You can
```

File: README.md [hunk 4]
```diff
@@ -124,14 +124,29 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
...
+turn this off by adding the following line to your `_config.local.yml` file:
```
Comment (hickskenh) [inline README.md]:
```suggestion
turn off link checking by adding the following line to your `_config.local.yml` file:
```

**Comment by filmaj** [comment]
Thanks so much for the suggestions @hickskenh, I've applied them all.

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
@filmaj Thank you for adding this useful information to our docs.  I have approved your PR and will merge after all tests are run.

File: README.md [hunk 5]
```diff
@@ -124,14 +124,26 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
 > ***TIP***
 > Leave the serve terminal open and running. Every time you save changes to a file, it automatically regenerates the site so you can test the output immediately. Changing the `_config.yml` file requires a fresh build. Using the `--incremental` option limits re-builds to posts and pages that have changed.
 
-### To minimize build time locally:
+### Minimizing Build Time
 
-You can speed the local build time up by only building one of the three versions that the full `rake preview` command builds. These are:
+#### Building Specifc Editions
```
Comment (dshevtsov) [inline README.md]:
```suggestion
#### Building specific editions
```

File: README.md [hunk 6]
```diff
@@ -124,14 +124,26 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
...
+
+You can speed the local build time up by only building one of the three versions that the full `rake preview` command builds. Building individual versions also enables incremental builds when you save a changed document, thus updating your preview. Incremental builds are disabled when `rake preview` is used to build all versions. To build a specific version, use one of the following commands:
 
 - `rake preview:ce`: Preview the Open Source edition locally
 - `rake preview:b2b`: Preview the B2B edition locally
 - `rake preview:ee`: Preview the Commerce edition locally
 
+#### Customizing Local Config
```
Comment (dshevtsov) [inline README.md]:
```suggestion
#### Customizing local config
```

File: README.md [hunk 7]
```diff
@@ -124,14 +124,26 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
...
+
+By creating your own `_config.local.yml` file, you can customize the [Jekyll config](https://jekyllrb.com/docs/configuration/options/) to suit your needs.
+
+##### Disable Link Checking
```
Comment (dshevtsov) [inline README.md]:
```suggestion
##### Disable link checking
```

File: README.md [hunk 8]
```diff
@@ -124,14 +124,26 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
...
+
+By default, `check_links` is enabled. This option verifies every link in every project file, which can significantly increase build times when iterating many changes quickly. You can turn off link checking by adding the following line to your `_config.local.yml` file:
+
+    `check_links: false`
```
Comment (dshevtsov) [inline README.md]:
````suggestion
```yaml
check_links: false
```
````

File: README.md [hunk 9]
```diff
@@ -124,14 +124,26 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
 > ***TIP***
 > Leave the serve terminal open and running. Every time you save changes to a file, it automatically regenerates the site so you can test the output immediately. Changing the `_config.yml` file requires a fresh build. Using the `--incremental` option limits re-builds to posts and pages that have changed.
 
-### To minimize build time locally:
+### Minimizing Build Time
```
Comment (dshevtsov) [inline README.md]:
```suggestion
### Minimizing build time
```

**Comment by filmaj** [comment]
Thanks @dshevtsov, applied your suggestions.

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
all tests have passed and this PR is approved. Merging and closing.

**Pull request merged**